src/components/action-popover/action-popover-menu.component.js

```diff
@@ -179,6 +189,8 @@ const ActionPopoverMenu = React.forwardRef(
         aria-labelledby={parentID}
         role="menu"
         ref={ref}
+        placement={placement}
+        buttonHeight={getButtonHeight(button)}
```

Comment:
`buttonHeight` isn't a prop  on the `Menu` component, we'll need to document it.

Reply:
Ah yeah good point I missed that

Reply:
The prop is internal to the `Menu` which is a `styled-component`